packages/kbn-management/settings/components/field_row/input_footer/change_image_link.tsx

```diff
@@ -70,7 +77,9 @@ export const ChangeImageLink = <T extends SettingType>({
               ariaLabel,
             },
           })}
-          onClick={() => onChange({ value: defaultValue })}
+          // We're using the value of undefined here to indicate that a person has cleared the
+          // image and has not yet chosen a new one.
+          onClick={() => onChange({ value: undefined })}
```

Comment:
It seems from the story that the "Change link" doesn't clear the current image:

https://github.com/elastic/kibana/assets/59341489/28ec29ca-4845-4363-9794-12dee0b2d823

Reply:
Fixed in latest.